describe new yaml field show_modal to skip textual input (#4240)
SumoLogic/sumologic-documentation: docs/cloud-soar/cloud-soar-integration-framework.md

   }
   end_point = '{incmanurl}/api/v2/incidents/{incidentid}'.format(incmanurl=args.incmanurl, incidentid=incidentID)
   session = requests.Session()
   session.verify = False
   additional_info = inc_det_after.get('additional_info')
   payload = {
      "additional_info": additional_info,
   }
   incident = session.put(end_point, headers=headers, data=payload, proxies=None, timeout=(5, 60))
   try:
   incident.raise_for_status()
   except Exception as e:
   sys.stderr.write(str(e))
   exit(0)

check_not_null_field: 'opt_92' #specify that the custom field with internal id “opt_92” should be populated in
order to display this button in the GUI.
hook:
 - incidentCustomAction #hook that generates a custom button in the incident tombstone when field opt_92 is not
null
```
For other example YAML files, see:
* [Trigger taskCustomAction definition file (Incident Tools)](/docs/cloud-soar/cloud-soar-integration-framework/#trigger-taskcustomaction-definition-file-incident-tools)
* [Trigger incidentCustomAction definition file (Incident Tools)](/docs/cloud-soar/cloud-soar-integration-framework/#trigger-incidentcustomaction-definition-file-incident-tools)

> #### show_modal (default true)
> 
> Specify whether textual input will be display 

Reviewer: ```suggestion
#### show_modal

Specifies whether to display the textual input. Default value is `true`. 
```
